fluent-langneg/test/langneg_test.js

```diff
@@ -111,11 +111,11 @@ const data = {
 const json = JSON.stringify;
 
 suite('Language Negotiation', () => {
+  for (const strategy in data) {
+    for (const groupName in data[strategy]) {
+      const group = data[strategy][groupName];
 
-  test('test suite matches', () => {
-    for (const strategy in data) {
-      for (const groupName in data[strategy]) {
-        const group = data[strategy][groupName];
+      test(`${strategy} - ${groupName}`, () => {
```

Comment:
I moved the `test` call into the nested for loop so that each case is run as a separate Mocha test, resulting in better test summary and error logging.